src/Pact/Analyze/Eval/Prop.hs

```diff
@@ -127,8 +139,12 @@ evalPropSpecific (Exists vid _name QTable prop) = do
   bools <- for (Map.keys tables) $ \tableName ->
     local (qeTableScope . at vid ?~ tableName) (evalProp prop)
   pure $ foldr (|||) true bools
-evalPropSpecific (Exists _vid _name (QColumnOf _tab) _p) =
-  throwErrorNoLoc "TODO: column quantification"
+evalPropSpecific (Exists vid _name (QColumnOf tabName) prop) = do
+  columns <- view (analyzeEnv . aeColumnIds . ix tabName)
+  bools <- for (Map.keys columns) $ \colName ->
+    let colName' = ColumnName $ T.unpack colName
+    in local (qeColumnScope . at vid ?~ colName') (evalProp prop)
+  pure $ foldr (|||) true bools
```

Comment:
does this need to be `foldr (|||) false bools`? otherwise it will always be true?

Reply:
very good point